packages/google_sign_in/google_sign_in_web/test/gapi_mocks/src/gapi_load.dart

```diff
@@ -0,0 +1,27 @@
+part of gapi_mocks;
+
+const String gapiInitSuccess = '''
+(function() {
+    function Gapi() {};
+    Gapi.prototype.load = function (script, callback) {
+        window.setTimeout(() => {
+          callback();
+        }, 30);
+    };
+
+    // Initialize the gapi.auth mock.
```

Comment:
Is this commented out code still needed?

Reply:
D'oh, not for this initial test, but for more complex ones I'll do. Let me remove it from here. Good catch!